ui/src/components/view/DedicateData.vue

```diff
@@ -121,8 +121,7 @@ export default {
       api('listDedicatedZones', {
         zoneid: this.resource.id
       }).then(response => {
-        if (response.listdedicatedzonesresponse.dedicatedzone &&
-            response.listdedicatedzonesresponse.dedicatedzone.length > 0) {
+        if (response.listdedicatedzonesresponse.dedicatedzone?.length > 0) {
```

Comment:
```suggestion
        if (response?.listdedicatedzonesresponse?.dedicatedzone?.length > 0) {
```

Reply:
@utchoang is the earlier approach wrong?
There is no '&&' operator elsewhere in the line above.

Reply:
@Yashikaone what's wrong?

Reply:
no I meant is this wrong?
if (response.listdedicatedzonesresponse.dedicatedzone?.length > 0)

Reply:
I was initially confused whether I should prefer the second approach or the first approach but I did not check whether ### response or ### response.listdedicatedzonesresponse is null or empty because the original codebase did not check it and I just wanted to make sure.